src/ValueResolver/NodeExpressionResolver.php

```diff
@@ -224,15 +224,14 @@ protected function resolveExprConstFetch(Expr\ConstFetch $node)
         $constantValue = null;
         $isResolved    = false;
 
-        /** @var ReflectionFileNamespace|null $fileNamespace */
-        $fileNamespace = null;
         $isFQNConstant = $node->name instanceof Node\Name\FullyQualified;
         $constantName  = $node->name->toString();
 
         if (!$isFQNConstant) {
             if (method_exists($this->context, 'getFileName')) {
                 $fileName      = $this->context->getFileName();
                 $namespaceName = $this->resolveScalarMagicConstNamespace();
+                /** @var ReflectionFileNamespace $fileNamespace */
```

Comment:
You can safely remove this, because IDE can detect type from `new ...` construct by itself :wink: